chirp/Cargo.toml

```diff
@@ -21,6 +21,7 @@ tokio = { version = "1", features = ["full"] }
 typeshare = "1"
 # url may conflict with reqwest::Url
 url = "2"
+labels = { path = "../../labels" }
```

Comment:
This is a relative path, don't you mean to use the URL `"https://github.com/exquisiteOntologist/labels"` ?

Reply:
hee hee I missed that, god I'm glad I have you to work with

Reply:
you're welcome